test/e2e/features/pageObjects/common/CaseManager.js

```diff
@@ -19,7 +19,7 @@ class CaseManager {
 
         this.continueBtn = new Button('button', 'Continue');
 
-        this.submitBtn = $("form button[@type = 'submit']");
+        this.submitBtn = element(by.xpath('//button[contains(text(),\'Submit\')]'));
```

Comment:
Services can update label of button in definitions, which will require update in our test and some other service may have different label. This change is more specific to the case being worked as part of this PR. Intention of this file to have more generic code that can be used for all services. And if we require any service specific changes they can be in their own file.  Can you make this change generic please.

Reply:
Hi @sreekanthpuligadda, this change is generic as this selector works for all buttons that contain Submit text, the previous selector didn't work anymore for submit buttons.